From 431e477b8e66ddbaba28a9f59fb8ae2e16d59cb5 Mon Sep 17 00:00:00 2001
From: Kay Sievers <kay@vrfy.org>
Date: Mon, 21 Oct 2013 03:49:03 +0200
Subject: [PATCH] rules: remove pointless MODE= settings

Changing the default MODE= for the group accessi, but not specifying
a GROUP= does not provide anything.

It disables the default logic that the mode switches to 0660 as soon
as a GROUP= is specifed, which make custom rules uneccesarily complicated.

https://bugs.freedesktop.org/show_bug.cgi?id=70665
---
 rules/50-udev-default.rules | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/rules/50-udev-default.rules b/rules/50-udev-default.rules
index f764789..679dfdf 100644
--- a/rules/50-udev-default.rules
+++ b/rules/50-udev-default.rules
@@ -20,9 +20,7 @@ KERNEL=="tty[A-Z]*[0-9]|pppox[0-9]*|ircomm[0-9]*|noz[0-9]*|rfcomm[0-9]*", GROUP=
 
 SUBSYSTEM=="mem", KERNEL=="mem|kmem|port", GROUP="kmem", MODE="0640"
 
-SUBSYSTEM=="input", KERNEL=="mouse*|mice|event*", MODE="0640"
-SUBSYSTEM=="input", KERNEL=="ts[0-9]*|uinput", MODE="0640"
-SUBSYSTEM=="input", KERNEL=="js[0-9]*", MODE="0644"
+SUBSYSTEM=="input", KERNEL=="js[0-9]*", MODE="0664"
 
 SUBSYSTEM=="video4linux", GROUP="video"
 SUBSYSTEM=="misc", KERNEL=="agpgart", GROUP="video"
@@ -63,7 +61,7 @@ SUBSYSTEM=="raw", KERNEL=="raw[0-9]*", GROUP="disk"
 SUBSYSTEM=="aoe", GROUP="disk", MODE="0220"
 SUBSYSTEM=="aoe", KERNEL=="err", MODE="0440"
 
-KERNEL=="rfkill", MODE="0644"
+KERNEL=="rfkill", MODE="0664"
 KERNEL=="tun", MODE="0666", OPTIONS+="static_node=net/tun"
 
 KERNEL=="fuse", MODE="0666", OPTIONS+="static_node=fuse"
-- 
1.8.4.652.g0d6e0ce

